### PR TITLE
Start of a Website

### PR DESCRIPTION
**Pull request by henri-tremblay**
This would be a start of a website served from the master branch. It requires some admin configuration explained here https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site including DNS routing to github.

Check `website.adoc` to run it locally. I'm not a web developer so it's ugly. But it works. One thing is that I use _data/champions.yml as the data file. Not the readme. So I'm not sure if we could script something similar in the readme to have all the data in one place.

### CODE REVIEW

**Comment by rokon12** [comment]
I ran it locally on my machine, it looks great. Should we also take into consideration the possibility of introducing search functionality ?

**Comment by kabutz** [comment]
I'd keep it as simple as possible. If all the names are already on one page, listed alphabetically by surname, then CTRL+F will find it quickly

**Comment by henri-tremblay** [comment]
Agreed. If someone wants to add a filter later, I will let him play.

**Comment by henri-tremblay** [comment]
So, are we merging? What's the next step?

**Comment by aalmiray** [comment]
The data file is missing an option to list all marks related to a JC (currently set to alumni, deceased, retired). Bear in mind that more than one mark may apply. We don't have that use case yet but it'll surely come. Also we might add other marks in the future. 

The late Alex Blewitt suggested the use of Mastodon/Matrix for himself instead of/additionally to Twitter. Other JCs may do the same. Thus having a explicitly listed column/attribute as "Twitter" may be a limitation. There's a PR with the discussion we had in this subject. 

I'm weary of moving all data to a YAML file instead of plain Markdown as we loose a quick glance/render of the data. OTOH there's no _structure_ in Markdown that we could instead build with YAML.

**Comment by henri-tremblay** [comment]
All fair points.

File: _data/champions.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- name: Henri Tremblay
+  image: images/avatars/henri_tremblay.png
+  link: https://blog.tremblay.pro
+  twitter: henri_tremblay
```
Comment (evanchooly) [inline _data/champions.yml:4]: subdocuments here would allow for an abitrary number of socials.

**Comment by aalmiray** [comment]
Thank you @henri-tremblay. I've followed on your suggestions and built a site using JBang + JBake. Will use icons from this PR.

**Comment by henri-tremblay** [comment]
Thanks! JBang + JBake is a better solution since it's a Java solution. But putting my hands in there was a bit too much. I'm glad you were able to.

**Pull request closed without merging**